### ISSUE TITLE
[Request] Backwards Compatibility with Klipper 11

### ISSUE THREAD

**Issue by f00d4tehg0dz** [moggieuk/Happy-Hare#426]
I am requesting to have the toolhead backwards compatible with Klipper 11. I and others cannot upgrade to Klipper 12 due to hardware constraints with our Flashforge Adventurer 5M series. 

The last commit that seems to work with Klipper 11 is https://github.com/moggieuk/Happy-Hare/tree/5d84cb25b83c494f0224929000f45e2a46678e21?tab=readme-ov-file#---installation  

However it lacks the new features and many bug fixes. 

**module 'toolhead' has no attribute 'BUFFER_TIME_HIGH'**

```MMU Hardware Initialization -------------------------------
Unhandled exception during connect
Traceback (most recent call last):
  File "/root/printer_software/klipper/klippy/klippy.py", line 175, in _connect
    self._read_config()
  File "/root/printer_software/klipper/klippy/klippy.py", line 141, in _read_config
    self.load_object(config, section_config.get_name(), None)
  File "/root/printer_software/klipper/klippy/klippy.py", line 130, in load_object
    self.objects[section] = init_func(config.getsection(section))
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/printer_software/klipper/klippy/extras/mmu.py", line 7791, in load_config
    return Mmu(config)
           ^^^^^^^^^^^
  File "/root/printer_software/klipper/klippy/extras/mmu.py", line 813, in __init__
    self._setup_mmu_hardware(config)
  File "/root/printer_software/klipper/klippy/extras/mmu.py", line 826, in _setup_mmu_hardware
    self.mmu_toolhead = MmuToolHead(config, self, self.homing_extruder)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/printer_software/klipper/klippy/extras/mmu_toolhead.py", line 44, in __init__
    self.move_queue.set_flush_time(toolhead.BUFFER_TIME_HIGH) # Happy Hare: Use base class (older klipper)
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'toolhead' has no attribute 'BUFFER_TIME_HIGH'
webhooks client 3034598608: New connection
webhooks client 3034598608: Client info {'program': 'Moonraker', 'version': 'v0.11.0-4-g4d24de080-AD5M-20240506'}```

**Comment by f00d4tehg0dz** [comment]
Disregard. I see you may have fixed it here: https://github.com/moggieuk/Happy-Hare/pull/388/commits

**Comment by moggieuk** [comment]
I think this is addressed and PR388 is incorporated into the new v2.7.1 release.

Early access is available with:
```
./install.sh -b v271beta
```

I will point out that I cannot support earlier version of klipper myself because of the impracticality of installing an older system (not just klippy but re-flashing all the mcus).  Thus I will rely on community support to validate it works.

**Comment by f00d4tehg0dz** [comment]
> I think this is addressed and PR388 is incorporated into the new v2.7.1 release.
> 
> Early access is available with:
> 
> ```
> ./install.sh -b v271beta
Thank you, and Happy-Hare works now with that beta branch. Thank you so much and the community!
